src/server/server.cpp

```diff
@@ -1081,7 +1081,7 @@ void server::handle_query(socket_ptr socket, simple_wml::node& query)
 		command == "motd" ||
 		command == "netstats" ||
 		command == "netstats all" ||
-		command == "player_version" ||
+		command.compare(0, 7, "version") == 0 ||
```

Comment:
Why are you using `.compare` here?

Reply:
becasue the request is not "version" but "version <playername>"

Reply:
But this'll also match anything nonsensical beginning with "version".